index.html

```diff
@@ -78,13 +77,65 @@
+                        <select class="language">
+                            <option value="en">english</option>
+                            <option value="ru">russian</option>
+                        </select>
+                    </div>
+                    <div class="change settings__photo">
+                        <p class="title title__photo">Change Photo Source</p>
+                        <select class="photo">
+                            <option value="github">GitHub</option>
+                            <option value="unsplash">Unsplash Api</option>
+                            <option value="flickr">Flickr Api</option>
+                        </select>
+                    </div>
+                </div>
+                <div class="settings__blocks">
+                    <div class="settings__blocks__wrapper">
+                        <div class="block">
+                            <input type="checkbox" id="time" role="switch" name="time" checked>
+                            <label for="time">time</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="date" role="switch" name="date" checked>
+                            <label for="date">date</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="greeting" role="switch" name="greeting" checked>
+                            <label for="greeting">greeting</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="quotes" role="switch" name="quotes" checked>
+                            <label for="quotes">quotes</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="weather" role="switch" name="weather" checked>
+                            <label for="weather">weather</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="audio-player" role="switch" name="audio-player" checked>
+                            <label for="audio-player">player</label>
+                        </div>
+                        <div class="block">
+                            <input type="checkbox" id="todo" role="switch" name="todo" checked>
+                            <label for="todo">todo</label>
+                        </div>
+                    </div>
+                </div>
+            </div>
             <div class="quotes">
                 <span class="icon qoutes-icon"></span>
                 <q class="quote"></q>
                 <cite class="author"></cite>
             </div>
         </div>
     </footer>
-    <script type="module" src="./src/index.js"></script>
+    <!-- <script type="module" src="./src/index.js"></script> -->
```

Comment:
also here, if you think that you don't need it, just remove, no need to comment out